src/devices/Ad7193/Ad7193.sln

```diff
@@ -0,0 +1,79 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
```

Comment:
We Usually don't add sln files to the repo. This might actually change in the future but at least for consistency lets keep this file out from your PR for now.

Reply:
Hmmm... I would definitelly keep them for the individual bindings. You'll need them anyway, and it helps keeping things together a lot. Most recent changes kept it IIRC.

Reply:
I don't believe new bindings have added solutions, at least not that I'm aware of, and if there are any is only because we missed them on the Pull Request. You may be right that perhaps adding them is the right thing to do, but I would rather do this as a bulk for all devices in order to always have consistency on the repo rather than needing to special case.